### PR TITLE
Stellar theme ready

### PR DESCRIPTION
**Pull request by pbrudny**
Remove all Gatsby files.
Add basic readme.
https://github.com/gatsbymanor/gatsby-theme-kit/issues/7

### CODE REVIEW

**Comment by nodox** [comment]
@pbrudny Hey this is awesome! 🥇  thanks for taking the time to work this out! I'll review this closer to the weekend! Glad you took the opportunity to better your front end skills! You rock! 😄

File: components/Main.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Main(props) {
+  return (
+    <div id="main">
+     { props.children }
```
Comment (nodox) [inline components/Main.jsx]: Nit: {props.children}

Comment (nodox) [reply components/Main.jsx]: Might fix that highlighted `div` below.

File: nightwatch.conf.js [hunk 2]
```diff
@@ -1,80 +0,0 @@
-// Helpful
```
Comment (nodox) [inline nightwatch.conf.js:1]: How come this file is being shown as deleted? I cloned your repo and the file is this there.

File: jest.config.js [hunk 3]
```diff
@@ -1,15 +0,0 @@
-module.exports = {
```
Comment (nodox) [inline jest.config.js:1]: How come this file is being shown as deleted? I cloned your repo and the file is this there.

File: package.json [hunk 4]
```diff
@@ -1,63 +0,0 @@
-{
```
Comment (nodox) [inline package.json:1]: How come this file is being shown as deleted? I cloned your repo and the file is this there. 

Another bizarre one.

File: pages/generic.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Footer from '../components/Footer';
+import picture from '../images/pic04.jpg';
+
+export default function Generic() {
+  return (
+  <div id="wrapper">
```
Comment (nodox) [inline pages/generic.js]: Nit: needs to go in 1 indent level

**Review by nodox — COMMENTED** [review]
@pbrudny Left some comments. I will give it another run through this weekend, so far its looking great! We can go through the merging process this weekend its a bit convoluted but its a great opportunity to improve the experience by increasing documentation! Thanks again 😄 👍

File: nightwatch.conf.js (hunk 2 above)
Comment (pbrudny) [reply nightwatch.conf.js:1]: @nodox I had 2 PR's so maybe you cloned from the first branch. On stellar_theme_branch I tried to remove everything aparat the 'src'. Sorry. Maybe I did it messy :)

**Comment by nodox** [comment]
You're going to want to make the PR on this [Stellar](https://github.com/gatsbymanor/gatsby-theme-stellar) repo. Then we will continue with the little bit of cleanup. @pbrudny

File: nightwatch.conf.js (hunk 2 above)
Comment (nodox) [reply nightwatch.conf.js:1]: Nope so far so good! 👍

**Pull request closed without merging**